src/m365/spo/commands/page/page-template-list.ts

```diff
@@ -0,0 +1,86 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { PageTemplateResponse } from './PageTemplateResponse';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+  webUrl: string;
+}
+
+class SpoPageControlListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.PAGE_TEMPLATE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists all page templates in the given site';
+  }
+
+  public defaultProperties(): string[] | undefined { 
+    return ['AbsoluteUrl', 'FileName', 'Id', 'PageLayoutType', 'Url']; 
```

Comment:
The output in the text mode is pretty long and contains multiple URLs:

<img width="915" alt="image" src="https://user-images.githubusercontent.com/11164679/104952066-12df0580-59c4-11eb-8614-d6d3bd5e38d7.png">

Since we know the URL of the current site, let's remove the AbsoluteUrl. It's still available in the JSON mode and can be overriden in the text mode using a JMESPath query if needed.

Reply:
Sure, sounds good to me.